guacamole-common-js/src/main/webapp/modules/BlobWriter.js

```diff
@@ -110,6 +110,10 @@ Guacamole.BlobWriter = function BlobWriter(stream) {
         var offset = 0;
         var reader = new FileReader();
 
+        var sleep = function sleep(ms) {
```

Comment:
I expect this function will simply disappear (see below), but for reference: all functions need to be documented. In this case, this would mean a JSDoc block comment.